src/styles/settings.scss

```diff
@@ -2,6 +2,7 @@
 $white: #fff;
 $elephant: #0F2B3B;
 $robinsEggBlue: #00BDB5;
+$osloGray: #838C93;
```

Comment:
`$robinsEggBlue`, `$osloGray`...? 
You have a hidden poet inside @kfilip94 😅

Reply:
Sadly it's not me :disappointed:, I use this site:
http://chir.ag/projects/name-that-color/#6195ED